addons/sys_rpc/fnc_handleData.sqf

```diff
@@ -1,16 +1,15 @@
 /*
  * Author: ACRE2Team
- * SHORT DESCRIPTION
+ * Handles the receipt of messages from the teamspeak plugin. Firstly by de-serializing the recieved message. It will then call any procedure handlers.
  *
  * Arguments:
- * 0: ARGUMENT ONE <TYPE>
- * 1: ARGUMENT TWO <TYPE>
+ * 0: Message <STRING>
  *
  * Return Value:
- * RETURN VALUE <TYPE>
+ * Always returns true <BOOL>
```

Comment:
This is not true, L22 has no return value. Should probably change that to return `false` and have return value as `* Successfully handled <BOOL>`